dask/values.yaml

```diff
@@ -97,6 +97,11 @@ jupyter:
   servicePort: 80
   # This hash corresponds to the password 'dask'
   password: 'sha1:aae8550c0a44:9507d45e087d5ee481a5ce9f4f16f37a0867318c'
+  persistent:
+    enable: false
+    storageClass: "default"
+    size: 1Gi
+    path: /home/jovyan/persistent
```

Comment:
I would recommend making this just `/home/jovyan`

Reply:
I was trying to explicitly not overwrite the examples directory with the pv. I see your point though, it would be much clearer to the user if the entire home directory is whats saved.

Reply:
The examples are created at runtime. So this should've be a problem.

Reply:
When I make the mount path `/home/jovyan` the examples do not show up.

Checking the Dockerfile from the `dask-docker` repo shows they are added at build time.

https://github.com/dask/dask-docker/blob/master/notebook/Dockerfile#L47

Reply:
Oh interesting. I must be thinking of the Pangeo helm chart.

Reply:
I'm wondering how you want to handle this. I can submit a PR to the docker-helm repo to copy the examples to another directory (say /usr/share/doc/dask-examples). That way they would be independent of the PV, and always available.

Reply:
Hey @jacobtomlinson how would you wan tot handle this?

Reply:
> I can submit a PR to the docker-helm repo to copy the examples to another directory (say /usr/share/doc/dask-examples).